src/Kovan-DssSpell.sol

```diff
@@ -16,36 +16,275 @@
 
-    address constant MCD_FLASH = 0x5aA1323f61D679E52a90120DFDA2ed1A76E4475A;
+    // Maker changelog 
+    address public constant MAKER_CHANGELOG = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
 
     function actions() public override {
-        // ---------------------------- Add MCD_FLASH ----------------------------
-        address MCD_VAT = DssExecLib.vat();
-        Fileable(MCD_FLASH).file("max", 500_000_000 * WAD);
-        Fileable(MCD_FLASH).file("toll", 5 * WAD / 10000);
-        DssExecLib.authorize(MCD_VAT, MCD_FLASH);
-        DssExecLib.setChangelogAddress("MCD_FLASH", MCD_FLASH);
-
-        // ---------------------------- Update Chainlog version ----------------------------
-        DssExecLib.setChangelogVersion("1.9.1");
+        // https://vote.makerdao.com/polling/QmRCn7Mh#poll-detail
+        CentrifugeCollateralValues memory RWA003 = CentrifugeCollateralValues({
+            MCD_JOIN: 0x4CCc7fED3912A32B6Cf7Db2FdA1554a9FF574099,
+            GEM: 0xDBC559F5058E593981C48f4f09fA34323df42d51,
+            OPERATOR: 0x45e17E350279a2f28243983053B634897BA03b64,
+            INPUT_CONDUIT: 0x45e17E350279a2f28243983053B634897BA03b64,
+            OUTPUT_CONDUIT: 0x45e17E350279a2f28243983053B634897BA03b64,
+            URN:  0x993c239179D6858769996bcAb5989ab2DF75913F,
+            gemID: "RWA003",
+            joinID: "MCD_JOIN_RWA003_A",
+            urnID: "RWA003_A_URN",
+            inputConduitID: "RWA003_A_INPUT_CONDUIT",
+            outputConduitID: "RWA003_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA003",
+            ilk: "RWA003-A",
+            ilk_string: "RWA003-A",
+            ilkRegistryName: "RWA003-A: Centrifuge: ConsolFreight",
+            RATE: SIX_PCT,
+            CEIL: 2 * MILLION,
+            PRICE: 2_359_560 * WAD,
+            MAT: 10_500,
+            TAU: 0,
+            DOC: "QmQMNfSbGS8qkJbatQgxMUsz27G8YELWgtXeLs8uFCZoY8"
+        });
+
+        // https://vote.makerdao.com/polling/QmRWg6nZ#poll-detail
+        CentrifugeCollateralValues memory RWA004 = CentrifugeCollateralValues({
+            MCD_JOIN: 0xa92D4082BabF785Ba02f9C419509B7d08f2ef271,
+            GEM: 0x146b0abaB80a60Bfa3b4fDDb5056bBcFa4f1fec1,
+            OPERATOR: 0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3,
+            INPUT_CONDUIT: 0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3,
+            OUTPUT_CONDUIT: 0x303dFE04Be5731207c5213FbB54488B3aD9B9FE3,
+            URN:  0xf22C7F5A2AecE1E85263e3cec522BDCD3e392B59,
+            gemID: "RWA004",
+            joinID: "MCD_JOIN_RWA004_A",
+            urnID: "RWA004_A_URN",
+            inputConduitID: "RWA004_A_INPUT_CONDUIT",
+            outputConduitID: "RWA004_A_OUTPUT_CONDUIT",
+            pipID: "PIP_RWA004",
+            ilk: "RWA004-A",
+            ilk_string: "RWA004-A",
+            ilkRegistryName: "RWA004-A: Centrifuge: Harbor Trade Credit",
+            RATE: SEVEN_PCT,
+            CEIL: 7 * MILLION,
+            PRICE: 8_815_730 * WAD,
```

Comment:
did I do my math wrong or is this line wrong?
```
7 * 10^6 * 1.07^2 * 1.10
8778000.00
```

Reply:
I get 8,815,730 using two different calculators, so I have pretty high confidence. Not sure what's up with your calculation--if using `bc`, maybe try some extra parentheses to be on the safe side.

Reply:
it was a rounding problem:
```
8815730.000000
```